Packs/ThreatExchange/README.md

```diff
@@ -0,0 +1,402 @@
+## Overview
+---
+
+Receive threat intelligence about applications, IP addresses, URLs and hashes, a service by Facebook
+This integration was integrated and tested with version xx of ThreatExchange
```

Comment:
update the version

Reply:
I dont think there is a version for it cause its a service by facebook

Reply:
they should have a version.. if not let remove this line